src/features/create-dao/steps/GovernanceForm.test.tsx

```diff
@@ -0,0 +1,195 @@
+		fireEvent.click(
+			screen.getByRole('button', {
+				name: 'Next',
+			}),
+		);
+
+		await waitFor(() =>
+			expect(
+				screen.getByText('The voting period field is required.'),
+			).not.toBeInTheDocument(),
+		);
+	});
+
+	test('should correctly validate required voting system field', async () => {
+		render(
+			<ZUIProvider>
+				<CreateDAOFormContext.Provider value={DEFAULT_PROVIDER_VALUES}>
+					<GovernanceForm {...DEFAULT_PROPS} />
+				</CreateDAOFormContext.Provider>
+			</ZUIProvider>,
+		);
+
+		fireEvent.blur(screen.getByPlaceholderText(/Select voting system.../i));
+		fireEvent.click(
+			screen.getByRole('button', {
+				name: 'Next',
+			}),
+		);
+
+		await waitFor(() =>
+			expect(
+				screen.getByText('The voting system field is required.'),
+			).not.toBeInTheDocument(),
+		);
+	});
+
+	test('should correctly validate required DAO token address field', async () => {
+		render(
+			<ZUIProvider>
+				<CreateDAOFormContext.Provider value={DEFAULT_PROVIDER_VALUES}>
+					<GovernanceForm {...DEFAULT_PROPS} />
+				</CreateDAOFormContext.Provider>
+			</ZUIProvider>,
+		);
+
+		fireEvent.blur(screen.getByPlaceholderText(/Enter DAO token address.../i));
+		fireEvent.click(
+			screen.getByRole('button', {
+				name: 'Next',
+			}),
+		);
+
+		await waitFor(() =>
+			expect(
+				screen.getByText('The DAO token address field is required.'),
+			).not.toBeInTheDocument(),
+		);
+	});
+
+	test('should correctly validate required voting threshold field', async () => {
+		render(
+			<ZUIProvider>
+				<CreateDAOFormContext.Provider value={DEFAULT_PROVIDER_VALUES}>
+					<GovernanceForm {...DEFAULT_PROPS} />
+				</CreateDAOFormContext.Provider>
+			</ZUIProvider>,
+		);
+
+		fireEvent.blur(screen.getByPlaceholderText(/Enter voting threshold.../i));
+		fireEvent.click(
+			screen.getByRole('button', {
+				name: 'Next',
+			}),
+		);
+
+		await waitFor(() =>
+			expect(
+				screen.getByText('The voting threshold field is required.'),
+			).not.toBeInTheDocument(),
+		);
+	});
+
+	test('should not fire onSubmit when next button clicked and field values are invalid', async () => {
+		render(
+			<ZUIProvider>
+				<CreateDAOFormContext.Provider value={DEFAULT_PROVIDER_VALUES}>
+					<GovernanceForm {...DEFAULT_PROPS} />
+				</CreateDAOFormContext.Provider>
+			</ZUIProvider>,
+		);
+
+		const user = userEvent.setup();
+		await user.click(screen.getByRole('button', { name: /Next/i }));
+
+		await waitFor(() => expect(onSubmit).not.toHaveBeenCalled());
+	});
+
+	test('should fire onSubmit with expected params when next button clicked and field values are valid', async () => {
+		render(
+			<ZUIProvider>
+				<CreateDAOFormContext.Provider value={DEFAULT_PROVIDER_VALUES}>
+					<GovernanceForm {...DEFAULT_PROPS} />
+				</CreateDAOFormContext.Provider>
+			</ZUIProvider>,
+		);
+
+		const user = userEvent.setup();
+
+		await user.click(screen.getByText(/MAJORITY/i));
```

Comment:
I think you could remove any use of `await` when using `getBy...`.

Reply:
The `await` is for the user.click action. Removing breaks the tests:

![Screen Shot 2022-11-25 at 1 07 30 PM](https://user-images.githubusercontent.com/13953984/203876626-1763bc54-a192-484f-b479-58419ed63ffb.png)